src/combinatorics/set_family/compression/uv.lean

```diff
@@ -97,12 +95,20 @@ lemma compress_of_disjoint_of_le (hua : disjoint u a) (hva : v ≤ a) :
   compress u v a = (a ⊔ u) \ v :=
 if_pos ⟨hua, hva⟩
 
+lemma compress_of_disjoint_of_le' (hva : disjoint v a) (hua : u ≤ a) :
+  compress u v ((a ⊔ v) \ u) = a :=
```

Comment:
Is this true as `compress u v (compress v u a) = a`?

Reply:
Nope.
```
#eval compress ({ff} : finset bool) ∅ (compress ∅ {ff} {tt}) -- {tt, ff}
```